appinfo/routes.php

```diff
@@ -32,6 +32,14 @@
 				'url' => '/accounts/{accountId}/folders/{folderId}/messages/{messageId}/toggleStar',
 				'verb' => 'POST'],
 			[
+				'name' => 'messages#toggleUnseen',
+				'url' => '/accounts/{accountId}/folders/{folderId}/messages/{messageId}/toggleUnseen',
+				'verb' => 'POST'],
+			[
+				'name' => 'messages#getUnseenCount',
+				'url' => '/accounts/{accountId}/folders/{folderId}/getUnseenCount',
+				'verb' => 'POST'],
```

Comment:
It's about the second route, which shouldn't change anything on the server (``messages#getUnseenCount``).